### PR TITLE
Add a function to upgrade poc receipts not serialized as protos

### PR DESCRIPTION
**Pull request by Vagabond**

### CODE REVIEW

File: src/transactions/v1/blockchain_poc_path_element_v1.erl [hunk 1]
```diff
@@ -16,7 +16,8 @@
     witnesses/1,
     print/1,
     json_type/0,
-    to_json/2
+    to_json/2,
+    maybe_upgrade/1
```
Comment (abhay) [inline src/transactions/v1/blockchain_poc_path_element_v1.erl:20]:
```suggestion
    maybe_proto/1
```

to be more explicit?

Comment (abhay) [reply src/transactions/v1/blockchain_poc_path_element_v1.erl:20]: ignore me. discussed offline.

**Review by abhay — APPROVED** [review]
Seems reasonable. Is there a matching miner PR?

**Review by andymck — APPROVED** [review]
lgtm, are you planning on calling upgrade from miner_poc_mgr:local_pocs/1?

**Comment by Vagabond** [comment]
> lgtm, are you planning on calling upgrade from miner_poc_mgr:local_pocs/1?

yes.

**Pull request merged**